app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+    before_save {self.email = email.downcase}
```

Comment:
indentation = 2 spaces
```suggestion
  before_save {self.email = email.downcase}
```